magnet_coils.py

```diff
@@ -23,18 +22,10 @@ def mesh_magnets(vol_ids, export_dir, logger):
 
     logger.info('Exporting coil mesh...')
     
-    # Define export paths
-    exo_path = Path(export_dir) / 'coil_mesh.exo'
-    h5m_path = Path(export_dir) / 'coil_mesh.h5m'
+    export_path = Path(export_dir) / 'coil_mesh.h5m'
     
-    # EXODUS export
-    cubit.cmd(f'export mesh "{exo_path}"')
-    
-    # Convert EXODUS to H5M
-    mb = core.Core()
-    exodus_set = mb.create_meshset()
-    mb.load_file(str(exo_path), exodus_set)
-    mb.write_file(str(h5m_path), [exodus_set])
+    # H5M export
+    cubit.cmd(f'export cf_dagmc "{export_path}" overwrite')
```

Comment:
I'm curious about the workflow here. Historically, a `dagmc` export was only for surface triangles and never volume mesh.  Is that changed for `cf_dagmc`?  Does this export both surface triangle info in a DAGMC-usable form as well as volume tetmesh?  Or just the mesh? How does it know?

Reply:
You're right, only the surface triangles get exported and not the full tetrahedra. I hadn't thought to look at the insides of the resultant mesh